src/main/java/in/projecteka/datanotificationsubscription/common/Constants.java

```diff
@@ -1,12 +1,20 @@
 package in.projecteka.datanotificationsubscription.common;
 
+import java.time.LocalDateTime;
+
 public class Constants {
     public static final String API_VERSION = "v1";
+    public static final String BLOCK_LIST = "blockList";
+    public static final String BLOCK_LIST_FORMAT = "%s:%s";
+    public static final LocalDateTime DEFAULT_CACHE_VALUE = LocalDateTime.MIN;
+
 
     //APIs
     public static final String CURRENT_VERSION = "/" + API_VERSION;
     public static final String PATH_HEARTBEAT = CURRENT_VERSION + "/heartbeat";
     public static final String HELLO_WORLD = CURRENT_VERSION + "/hello";
+    public static final String PATH_SUBSCRIPTION_REQUEST_SUBSCRIBE = CURRENT_VERSION + "/subscriptions/subscribe";
```

Comment:
I think @angshu has updated the URLs recently. Please verify with the updated PR [here](https://github.com/ProjectEKA/projecteka.github.io/pull/43/files)